WatchDogEclipsePlugin/WatchDog/build.properties

```diff
@@ -2,22 +2,20 @@ source.. = src/
 bin.includes = META-INF/,\
                .,\
                resources/,\
-               libs/joda-time-2.3.jar,\
-               libs/gson-2.2.4.jar,\
-               libs/httpclient-4.3.3.jar,\
-               libs/httpcore-4.3.2.jar,\
-               libs/commons-codec-1.6.jar,\
-               libs/commons-logging-1.1.3.jar,\
-               libs/fluent-hc-4.3.3.jar,\
-               libs/httpclient-cache-4.3.3.jar,\
-               libs/httpmime-4.3.3.jar,\
-               libs/commons-validator-1.4.0.jar,\
-               libs/mapdb-1.0.5.jar,\
-               libs/java-util-1.12.0.jar,\
+               lib/gson-2.2.4.jar,\
+               lib/httpclient-4.3.3.jar,\
+               lib/httpcore-4.3.2.jar,\
+               lib/commons-codec-1.6.jar,\
+               lib/commons-logging-1.1.3.jar,\
+               lib/fluent-hc-4.3.3.jar,\
+               lib/httpclient-cache-4.3.3.jar,\
+               lib/httpmime-4.3.3.jar,\
+               lib/commons-validator-1.4.0.jar,\
+               lib/mapdb-1.0.5.jar,\
                pom.xml,\
                plugin.xml,\
-               libs/jcommon-1.0.23.jar,\
-               libs/jfreechart-1.0.19-swt.jar,\
-               libs/jfreechart-1.0.19.jar,\
-               libs/commons-lang-2.6.jar
+               lib/jcommon-1.0.23.jar,\

```

Comment:
There's no standard, but you may remember that we mentioned this once- IntelliJ will (in some cases) use lib/ folder as default for looking for Jars, and you said to make it consistent in all the projects.